app/controllers/api/projects.php

```diff
@@ -83,6 +85,8 @@
             throw new Exception("'console' is a reserved project.", 400, Exception::PROJECT_RESERVED_PROJECT);
         }
 
+        [$dbForProject, $returnDB, $dbName] = $dbPool->getAnyFromPool($cache);
```

Comment:
Not sure I fully understand what happens here.

Reply:
This is the `Create Project` endpoint. Here we get a random database instance from the list of available database pools. 

`$dbForProject` - an instance of `Utopia\Database\Database`
`$returnDB` - a callback function that allows us to return the database to the pool
`$dbName` - we need to store the database's name in the console DB